### DISCUSSION TITLE
Tasmota autodiscovery home assistant problem

### DISCUSSION THREAD

**Discussion by zerlinux** [arendst/Tasmota#12606] [Q&A] [answered]
Hello everyone and sorry for the english ... I added some esp8266 devices with fw 9.5 to home assistant via tasmota integration (setoption19 off) but the entities are not detected. Any suggestions? Thank you

**Comment by ascillato** [comment, marked as answer]
Sorry, but without information we can't help you.

Have you followed the instructions on the docs https://tasmota.github.io/docs/Home-Assistant/ ?
Have you configured a MQTT server?
Have you search for your devices inside Home Assistant at Integrations/Tasmota ?
Have you searched in old issues and in the Chat?
What do you see in the Tasmota console?
Which devices do you have? Saying ESP8266 is too much generic. Have you configured relays, switches, etc.?
Which Home Assistant version do you have?

**Reply by zerlinux** [reply]
Hi, sorry for missing information. Esp8266 device wemos d1 Fw 9.5 home assistant 2021.7.1 The mqtt broker is set up and running, 10 more tasmota devices work fine. This wemos communicates with the broker, I can see it from the logs in the home assistant. It is configured with a switch and relative relay as a test. The Tasmota integration recognizes the provision but renders the entities unavailable. I also tried with other devices like nodemcu, esp01 and other wemos. I always encounter the same problem, entities are unavailable. Thank you

**Reply by ascillato** [reply]
Hi,

Sorry again but without information we can't help. Sorry.

Any logs?

Tasmota console with weblog 4
Mqtt logs?
Home assistant logs?
Steps you have done?
Configurations?
Status 0?

**Reply by zerlinux** [reply]
Ok, shortly after the screenshots of what you asked me, sorry but I'm not an expert. Thanks so much

**Reply by ascillato** [reply]
Please, provide the console output of `status 0` command of at least 2 devices.

**Reply by ascillato** [reply]
Have you checked that the *mqtt client name* is different than the *mqtt topic*?

**Reply by ascillato** [reply]
Do you have any console log when this happens? (With weblog set to 4)
And when your device reboots.

**Reply by t-anjan** [reply]
@ascillato - I know the original author of this discussion has not got back, but his problem sounds very similar to what I am facing. I had posted about this on the Home Assistant forum [here](https://community.home-assistant.io/t/tasmota-integration-not-detecting-devices/326083). I think my post has all the information you are asking for. Could you please let me know what the issue could be?

**Reply by ascillato** [reply]
@t-anjan

Please, open a new issue. Thanks.

**Reply by t-anjan** [reply]
@ascillato - done. [Here](https://github.com/arendst/Tasmota/discussions/12817).
